src/org/ods/component/RolloutOpenShiftDeploymentStage.groovy

```diff
@@ -240,8 +277,33 @@ class RolloutOpenShiftDeploymentStage extends Stage {
                     originalVersion = originalVersions[resourceKind][resourceName] ?: 0
                 }
 
-                def podData = rolloutDeployment(resourceKind, resourceName, originalVersion)
-
+                def podData = [:]
+                if (!isHelm) {
```

Comment:
See above, please provide a clean separation of concerns. Support and maintenance will be improved.